src/arbeidssøkerskjema/steg/3-Kvittering.tsx

```diff
@@ -20,7 +31,7 @@ const Kvittering: React.FC = () => {
       skjema?.innsendingsdato ? skjema.innsendingsdato : dagensDato
     )} `;
 
-  return skjema?.innsendingsdato ? (
+  return skjema?.innsendingsdato || true ? (
```

Comment:
`skjema?.innsendingsdato || true` <--

Hva ønsker vi å oppnå her? Blir den alltid `true` ?

Reply:
Oi ops, denne glemte jeg å fjerne! Var egentlig bare for lokal utvikling 🙈  Bra sett! Fjerner!